app/controllers/admin/userpage_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::UserpageController < ApplicationController
+  before_action :require_admin
+
+  include Admin::AdminCommonController
+
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts.created_at_this_month(Time.zone.today)
+    render template: 'mypage/show'
```

Comment:
[ASK] `template: ` は必要そうでした?
```suggestion
    render 'mypage/show'
```

Reply:
おそらく調べたら`template:`と書く書き方が出てきたのでそうしたのだと思いますが、なくても動いたので、消しました→[fc23138](https://github.com/tomo-ogawa-andpad/my_benkyo_rails_app/pull/1/commits/fc23138e12d0da194de530cfbe4c1e1481d17a32)

Reply:
よいですね、複数のやり方が考えられてその適切性に迷ったときは公式(もしくはそれに準ずる)ドキュメントを参照するのがよいプラクティスです。
以下を参照すると、「template:オプションを使うこともできます （なおRails 2.2以前はtemplate:を省略できませんでした）。」とあるので、二者で動作は変わらずで template は古い書き方であるということがわかります。
https://railsguides.jp/layouts_and_rendering.html#render%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89%E3%82%92%E4%BD%BF%E3%81%86